src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/SystemCommandLine/VerbParsers.tt

```diff
@@ -65,21 +68,24 @@ foreach (XElement verb in allElements)
         // Look for options directly under each noun
         GenerateAddOptionOrArgument(noun, nounFormalNameVar, 12);
 
-        bool handlerGenerated = GenerateSetCustomHandler(nounFormalNameVar, noun, 12);
-
-        if (!handlerGenerated)
+        // Generate handler for noun
+        PushIndent(Indent(12));
+        if (hasHustomHandler)
         {
-            // Generate handler for noun
-            GenerateCommandHandler(nounFormalNameVar, commandFormalName + nounFormalName, noun, 12);
+            GenerateSetCustomHandler(nounFormalNameVar, customHandlerName, options);
+        }
+        else
+        {
+            string binderClassName = GetCustomBinderClassName(verb, noun);
+            GenerateCommandHandlerCustomBinder(nounFormalNameVar, commandFormalName + nounFormalName, binderClassName, options);
         }
-#>
-
-            <#= commandFormalNameVar #>.AddCommand(<#= nounFormalNameVar #>);
-<#
 
+        WriteLine(string.Empty);
+        WriteLine($"{commandFormalNameVar}.AddCommand({nounFormalNameVar});");
+        PopIndent();
     } // end noun foreach
 
-    GenerateSetCustomHandler(commandFormalNameVar, verb, 12);
+    // TODO: Generate handler for commands
```

Comment:
**Note**: This will be addressed when implementing commands without verb (add, delete, list), such as `push` and others

Reply:
Please add a link to the GitHub issue for this TODO in the comment. TODO comments without linked issues can get forgotten.

Reply:
Ok. Fixed